### PR TITLE
The Families gamemode now supports admin policies.

### PR DESCRIPTION
**Pull request by Iamgoofball**
Requested by the headmins.
:cl:
admin: Admins can now set custom gamemode policy for Gangsters and Undercover Cops.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
what the hell is up with travis right now

**Comment by TiviPlus** [comment]
code/modules/antagonists/ert/ert.dm:231:error: undercover_cop.current: undefined var
code/game/gamemodes/gang/gang_things.dm:34:error: gangbanger.current: undefined var

**Comment by EgorDinamit** [comment]
You forgot Defines?

File: code/modules/antagonists/ert/ert.dm [hunk 1]
```diff
@@ -224,6 +224,11 @@
 	missiondesc += "<BR> <B>3.</B> If Security is found to be violating the rights of citizens, detain them as per your authority as Spinward Stellar Coalition officers."
 	missiondesc += "<BR> <B>4.</B> If the situation demands it, evacuate the station. Otherwise, remain on station and keep the peace."
 	to_chat(owner,missiondesc)
+	var/policy = get_policy(ROLE_FAMILIES)
+	if(policy)
+		to_chat(owner, policy)
```
Comment (ATH1909) [inline code/modules/antagonists/ert/ert.dm]: are you sure that the police should be getting the same policy messages as gangsters?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/antagonists/ert/ert.dm]: was this intentionally resolved? It looks like nothing changed and there's no comment here to suggest it's intended.

**Comment by ExcessiveUseOfCobblestone** [comment]
Waiting for a response on a resolved comment I have now unresolved.

File: code/modules/antagonists/ert/ert.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/antagonists/ert/ert.dm]: Intentional, my bad. Forgot to actually hit the comment button. Due to how gamemodes work we only have one ROLE_X for families.

**Comment by Iamgoofball** [comment]
re-resolved it

**Pull request merged**